infra/terraform/global/main.tf

```diff
@@ -34,17 +34,27 @@ module "aws_account_logging" {
   vpcflowlogs_s3_bucket_name = var.vpcflowlogs_s3_bucket_name
 
   vpc_id = var.vpc_id
+  tags   = local.tags
 }
 
 module "mojanalytics_concourse_iam_list_roles_user" {
   source      = "./modules/iam_list_roles"
   org_name    = "mojanalytics"
   system_name = "concourse"
+  tags        = local.tags
 }
 
 module "ses_domain" {
-  source = "./modules/ses_domain"
-  domain = var.platform_root_domain
-
+  source              = "./modules/ses_domain"
+  domain              = var.platform_root_domain
   aws_route53_zone_id = aws_route53_zone.platform_zone.zone_id
 }
+
+locals {
+  tags = {
+    business-unit = "Platforms"
+    application   = "analytical-platform"
+    owner         = "analytical-platform:analytics-platform-tech@digital.justice.gov.uk"
+    is-production = "true"
```

Comment:
Should `is-production` always be `"true"`?

Several resources also have the `env` tag (which currently can be `env="dev"` or `env="alpha"`).

You could have a look at the `platform` variables to have an idea of what I mean.

Reply:
Yeah, good point. This really does need to be a switch. I should be able to pick this up when it's applied.

Reply:
On second thoughts. I've just reread the code. There is only one workspace `default`, and all of the resources that are being tagged here are actually production in some way.